app/services/notification_service.rb

```diff
@@ -1,9 +1,39 @@
 class NotificationService
   DEFAULT_LOCALE = :en
 
-  attr_reader :response, :error
-  def client
-    @client ||= Twilio::REST::Client.new(twilio_account_sid, twilio_auth_token)
+  attr_reader :client
+  attr_reader :error
+  attr_reader :response
+  attr_reader :twilio_sender_number
+
+  def initialize
```

Comment:
I think this would be simpler written as:
```
  attr_reader :error
  attr_reader :response
  attr_reader :twilio_sender_number

  def initialize
    @test_mode = !SimpleServer.env.production?
    @twilio_sender_number = ENV.fetch("TWILIO_PHONE_NUMBER")

    if @test_mode
      @twilio_account_sid = ENV.fetch("TWILIO_TEST_ACCOUNT_SID")
      @twilio_auth_token = ENV.fetch("TWILIO_TEST_AUTH_TOKEN")
    else
      @twilio_account_sid = ENV.fetch("TWILIO_ACCOUNT_SID")
      @twilio_auth_token = ENV.fetch("TWILIO_AUTH_TOKEN")
    end
  end

  def test_mode?
    @test_mode
  end

  def client
    @client ||= Twilio::REST::Client.new(@twilio_account_sid, @twilio_auth_token)
  end
```